categorysite.html

```diff
@@ -66,117 +66,116 @@ <h4>Varukorg</h4>
 			</div>
 		</nav>
 <!-- header end -->
-    <div class="container col-12 col-md-6 flex-column">
-    	<div class="row">
+    <div class="container col-sm-12 col-md-12 d-flex flex-nowrap">
+    	<div class="col-md-7">
     	<content class="content">
 			<section class="Matkassar">
```

Comment:
Första wrappern måste innehålla `col-` klasserna, annars kommer inte layouten fungera bra.
Så antingen anger ni detta direkt här 

```suggestion
			<section class="Matkassar col-12">
```

Eller så raderar ni denna section och låter diven nedan vara den första wrappern.


**Viktigt att ni gör samma ändringar för de 3 resterande kategorierna nedan!!!!**